src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj

```diff
@@ -6,17 +6,19 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Blueprint</PackageId>
     <Description>Provides Microsoft Azure Blueprint management operations including the ability to create, update and delete Blueprint definition and assignment.</Description>
-    <Version>0.9.1-Preview</Version>
+    <Version>0.10.0-Preview</Version>
```

Comment:
Since this is a new API version, the Nuget major version needs to be bumped too.

Reply:
@filizt As we discussed offline, please bump this version to `1.8.0-preview` (please note the casing)

Reply:
@dsgouda after out email thread, updating the version as 0.10.0-preview